desktop/renderer-app/src/pages/CloudStoragePage/store.tsx

```diff
@@ -278,6 +278,37 @@ export class CloudStorageStore extends CloudStorageStoreBase {
         this.uploadTaskManager.addTasks(Array.from(files));
     }
 
+    public fetchMoreCloudStorageData = async (page: number): Promise<void> => {
+        const { files: cloudFiles } = await listFiles({
```

Comment:
Maintain a fetching state so that subsequent requests can be ignored during fetching state